client/src/reducers/user.js

```diff
@@ -18,18 +19,22 @@ export default function user(state = initialState, action) {
     switch (action.type) {
         case REQUEST_USER_INFO:
             return Object.assign({}, state, {
-                isRequesting: true,
+                isRequesting: true
             });
 
         case GET_USER_SUCCESS:
             return Object.assign({}, state, {
                 isRequesting: false,
                 user: action.payload.user,
-                code: action.payload.code,
+                code: action.payload.code
             });
         case GET_USER_FAILURE:
             return Object.assign({}, state, {
-                isRequesting: false,
+                isRequesting: false
+            });
+        case SEND_USER:
```

Comment:
I don't really understand this SEND_USER. Can't you get the user id from the state.user.user.id?